x-pack/test/reporting/services/reporting_api.ts

```diff
@@ -5,28 +5,57 @@
  */
 
 import expect from '@kbn/expect';
+// @ts-ignore no module definition
 import { indexTimestamp } from '../../../legacy/plugins/reporting/server/lib/esqueue/helpers/index_timestamp';
+import { FtrProviderContext } from '../ftr_provider_context';
 
-function removeWhitespace(str) {
+interface PDFAppCounts {
+  app: {
+    [appName: string]: number;
+  };
+  layout: {
+    [layoutType: string]: number;
+  };
+}
+
+export interface ReportingUsageStats {
+  available: boolean;
+  enabled: boolean;
+  total: number;
+  last_7_days: {
+    total: number;
+    printable_pdf: PDFAppCounts;
+    [jobType: string]: any;
+  };
+  printable_pdf: PDFAppCounts;
+  status: any;
+  [jobType: string]: any;
+}
+
+interface UsageStats {
+  reporting: ReportingUsageStats;
```

Comment:
I'd like to use the `ReportingUsageType` interface that is exposed in the Reporting plugin code. However, the structure returned by the service API endpoint is has some post-processing that converts all the field names to snake_case. It's a separate issue that requires cleanup.

Reply:
Filed an issue: https://github.com/elastic/kibana/issues/65338